docs/markdown/Adding-new-projects-to-wrapdb.md

```diff
@@ -168,77 +101,41 @@ error. For further details see [the main Wrap
 manual](Wrap-dependency-system-manual.md).
 
 Now you can create the build files and work on them until the project
-builds correctly.
+builds correctly. Remember that all files go in the directory
+`subprojects/packagefiles/<project-name>`.
 
 ```
 ${EDITOR} meson.build meson_options.txt
 ```
 
+The `wrap-file` section must contain a `patch_directory` property
```

Comment:
Nitpicking: `patch_directory` is not required if upstream has meson.build files already.